packages/bootstrap/scss/scrollview/_layout.scss

```diff
@@ -1 +1,3 @@
-@import "~@progress/kendo-theme-default/scss/scrollview/layout";
\ No newline at end of file
+$scrollview-pager-item-border-width: 0 !default;
```

Comment:
this, along with the rest of the variables, should be extracted in the `_variables.scss`